Felienne/hedy
# [UI IMPROVEMENT] Always shows tabs, also when in developer's mode

**Description**
In this PR we fix that the adventure tabs are always shown, also when in developers mode. This way users are still able to switch adventures, store programs within specific adventures and/or make the quiz. This is currently not possible and can be cumbersome when the forced developers mode is set in class customizations.

**Fixes**
This PR fixes #2546.

**How to test**
Verify that the tabs are still shown when entering developer's mode. Also verify that the switching is still working and that the developer's mode is automatically turned off when entering the quiz tab.
If you're unsure about any of these, don't hesitate to ask. We're here to help!
